oak/server/oak_node.h

```diff
@@ -54,12 +55,15 @@ class OakNode final : public Application::Service {
 
   void InitEnvironment(wabt::interp::Environment* env);
 
-  // Native implementation of the `oak.channel_read` host function.
+  // Native implementation of the `channel_read` host function.
```

Comment:
I think `oak.channel_read` is the correct (fully qualified) name of the host function?

Reply:
Fixed.